<?xml version="1.0" encoding="utf-8"?>

<launch>

    <node
        name="deepsort_tracker"
        pkg="deepsort_tracker"
        type="run_tracker.py"
        output="screen">
        
        <param name="config_file" value="deepsort_tracker.yaml" />
        <param name="deepsort_params" value="deepsort_parameters.yaml" />

    </node>

</launch>